GliaWidgets/ViewController/Survey/Components/ButtonView/Survey.ButtonView.swift

```diff
@@ -11,6 +11,18 @@ extension Survey {
             let title: String
             let state: State
             let onSelection: () -> Void
+
+            var accessibility: Accessibility {
```

Comment:
Still has embedded constants. We should passed the from outside in `Accessibility` struct.

Reply:
Haven't done it(

Reply:
Based on the discussion we had regarding this PR, we will rework how Props are passes in separate PR and then introduce `Accessibility` struct. I'm resolving this comment.